libs/prelude/Prelude/Applicative.idr

```diff
@@ -12,17 +12,17 @@ import Prelude.Functor
 
 ---- Applicative functors/Idioms
 
-infixl 2 <*>
+infixl 3 <*>
```

Comment:
It makes sense for the alternative operator to have lower precedence than an operator that might be used to create an alternative. `pa *> a <|> pb *> b` meaning `(pa *> a) <|> (pb *> b)` seems to make more sense intuitively than `p1 <|> p2 *> x` meaning `(p1 <|> p2) *> x`. Also, it's this way in Haskell.